app/models/user.rb

```diff
@@ -49,6 +49,11 @@ def to_s
     "#{first_name} #{last_name} <#{email}>"
   end
 
+  # accepts array or single organisation
+  def remove_access_to(organisations_to_remove)
+    self.organisations = self.organisations - [*organisations_to_remove]
```

Comment:
This is just golfing, but if you're trying to ensure that `organisations_to_remove` is an Array here, I think `Array(organisations_to_remove)` is more idiomatic.

Reply:
done, I saw both and picked one at random :-) I think yours is clearer in intent now I think about it